README.md

```diff
@@ -2,58 +2,88 @@
 
 # Project Name
 
-> Description the project.
-
+> Personal Portfolio project
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML/CSS
+- NPM, Git, Github
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+**This project includes linters for Reporting errors in HTML, CSS and An open-source, automated tool for improving the quality of web pages, which has audits for performance, accessibility, progressive web apps, SEO and more. It is a summary of all my works and known technologies**
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+1. Node package Manager(NPM)
+   - To install NPM on your local machine, please visit [ NPM official website](https://nodejs.org/en/download/)
+
+- Text editor(Vscode, Atom, NotePad++, Sublime)
+
 ### Setup
 
+Set up folder as:
+
+/Portfolio-project
+-.github/workflows - linters.yml
+
+- Package.json
+- index.html
+- /CSS
+  - style.css
+- /img
+  -/icons
+  - angellist.svg
+  - circle 2 header.svg
+  - down-arrow.svg
+  - email.png
+  - github.svg
+  - hamburger.svg
+  - linkedIn.svg
+  - medium.svg
+  - twitter.svg
+  - vector.svg
+  - work.svg
+    shapes.png
+- .stylelintrc.json
+- .hintrc
+
 ### Install
 
-### Usage
+Run the following commands to install dependencies
 
-### Run tests
+```
 
-### Deployment
+npm install --save-dev hint@7.x
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
 
+```
 
+### Usage
 
-## Authors
+```
 
-👤 **Author1**
+npx hint .
+npx stylelint "**/*.{css,scss}"
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+```
 
-👤 **Author2**
+### Deployment
+
+## Authors
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Onate Makoji David**
+
+- GitHub: [@githubhandle](https://github.com/aceDavon)
+- Twitter: [@twitterhandle](https://twitter.com/Sharkleshevon)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/david-makoji-b6090971)
 
 ## 🤝 Contributing
 
+- GitHub: [@githubhandle](https://github.com/KebebewAbabu)
+
```

Comment:
You can now remove the contributions you added from trials. Since this is just a review project, you might want to keep things simple with minimal files and links.